src/Inkluzitron/Models/BdsmTestOrgApi/ResultItem.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Newtonsoft.Json;
+
+namespace Inkluzitron.Models.BdsmTestOrgApi
+{
+    public class ResultItem
+    {
+
+        [JsonProperty("id")]
```

Comment:
Tu u těch properties by nemuselo být špatný nastavit povinnost (JsonProperty konstruktor na to má nějaký parametr), ať se případně rejectne divná odpověď s prázdným/null Id/Name.

Reply:
Ošetřeno v b9cf20a